### ISSUE TITLE
Add ingress category to civo marketplace

### ISSUE THREAD

**Issue by milindchawre** [civo/kubernetes-marketplace#249]
Add separate category for **ingress** in civo marketplace.

Putting all ingress controllers in same category make it easy:
- to detect different available options, right now these are scattered across different categories.
- to [prevent mutual exclusive controllers getting installed](https://github.com/civo/kubernetes-marketplace/issues/248).

**Comment by milindchawre** [comment]
Ping @andyjeffries @saiyam1814

**Comment by andyjeffries** [comment]
Sounds reasonable to me, but @saiyam1814 is running the marketplace so it's his call :-)

**Comment by saiyam1814** [comment]
Agree, we have a reasonable number of Ingress Controllers on Civo Marketplace so I agree to have a separate section for Ingress. We have 6 and one more PR raised for Kond by @milindchawre

**Comment by milindchawre** [comment]
It's been a while I have revisited this issue. It seems all the ingress controllers apps are under architecture category and it has got quiet crowded.

<img width="1498" alt="image" src="https://github.com/civo/kubernetes-marketplace/assets/21288765/6e05c978-a631-43e8-be78-4fc8b4e48afa">

**Comment by yashpawarrr** [comment]
Hello and Good evening to all the civo community members , I've heard about civo via social media platforms and it seems pretty intresting to me ...

**ABOUT ME** I am a second year undergrad. with knowledge of cloud , containerization and k8s along with backend development , I wanna implement my learning to real system may you please guide me how to take a start in the CIVO community ( I really appreciate your interest in this topic ) Thanks .
